roles/packages/tasks/main.yml

```diff
@@ -34,6 +34,7 @@
     - libyaml-dev
     - libxml2-dev
     - libxslt-dev
+    - libgmp3-dev
     - autoconf
```

Comment:
We'll need to add these dependencies into the vars files on the centos branch. I'll file a ticket.